back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,354 @@
+
+A tooltip is available: "Password to use to connect to your IMAP server."
+
+**Delete messages:** If this toggle is enabled, messages on the server will be deleted as soon as PrestaShop has retrieved them. 
+
+A tooltip is available: "Delete messages after synchronization. If you do not enable this option, the synchronization will take more time."
+
+**Create new threads:** If this toggle is enabled, this will create new threads for unrecognized e-mails.
+
+A tooltip is available: "Create new threads for unrecognized emails."
+
+**IMAP options (/pop3):** If this toggle is enabled, POP3 will be used instead of IMAP.
+
+A tooltip is available: "Use POP3 instead of IMAP."
+
+**IMAP options (/norsh):** If this toggle is enabled, the connection to the email server will not pre-authenticated.
+
+A tooltip is available: "Do not use RSH or SSH to establish a preauthenticated IMAP sessions."
+
+**IMAP options (/ssl):**  If this toggle is enabled, the connection to the email server will not be encrypted. 
+
+A tooltip is available: "Use the Secure Socket Layer (TLS/SSL) to encrypt the session."
+
+**IMAP options (/validate-cert):** If this toggle is enabled, PrestaShop will force the validation of the server's TLS/SSL certificate.
+
+A tooltip is available: "Validate certificates from the TLS/SSL server."
+
+**IMAP options (/novalidate-cert):** If this toggle is enabled, PrestaShop will never try to validate the server's TLS/SSL certificate. This is essential for servers with self-signed certificates.
+
+A tooltip is available: "Do not validate certificates from the TLS/SSL server. This is only needed if a server uses self-signed certificates."
+
+**IMAP options (/tls):** If this toggle is enabled, PrestaShop will force the use of StartTLS to encrypt the connection. Servers that do not support StartTLS will be rejected.
+
+A tooltip is available: "Force use of start-TLS to encrypt the seesion, and reject connection to servers that do not support it."
+
+**IMAP options (/notls)** If this toggle is enabled, PrestaShop will not use StartTLS to encrypt the session, even with servers that support it.
+
+A tooltip is available: "Do not user start-TLS to encrypt the session, even with servers that support it."
+
+### Multistore
+
+In a multistore context, the options listed above have [checkboxes](https://github.com/PrestaShop/PrestaShop/issues/19378) and [specific settings drop-down](https://github.com/PrestaShop/PrestaShop/issues/19330), take a look at the conditions and specs [here](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/multistoregeneralspecs.md)
+
+### SAVE BUTTON
+
+When clicking on save, if all settings have well been saved, then a successfull message is displayed: "The settings have been successfully updated."
+
+If all IMAP fields are not filled and if you click on save, then the following information message is displayed: "IMAP configuration is not correct"
```

Comment:
Is it blocking to save the configuration? Should it be a warning/error notification instead?

Reply:
It's not blocking to save the configuration, a message "The settings have been successfully updated." is displayed but the user is warned that the configuration is not correct

Reply:
In other words, the user can save an incorrect IMAP configuration? I feel like this behavior could be improved but I don't know how problematic it can be. In my opinion, we should turn this information notification into a warning one, or at least rephrase it to emphasize the fact that the configuration saved must not be left invalid.

Reply:
Yes, the wording here is not really correct as it's not that the configuration is not correct but rather incomplete
Unless this message shows up in some other case that I haven't identified, but from what I see it shows until all IMAP fields are filled out.

Reply:
I've looked into the documentation and it looks like the first 4 fields need to be filled to run the feature. So I suggest changing the behavior: IMAP URL, port, user, and password fields must be filled to be saved.

If the user tries to save an incomplete configuration, display an error notification: _Unable to save settings, please fill in all IMAP fields to configure your customer service options._ in _Admin.Catalog.Notification_.

If the user saves an incorrect configuration, display a warning notification (and not an information notification), cf. line 360 below.

@marionf, @Julievrz, what do you think?

Reply:
@LouiseBonnard Is imap not being installed on the server considered as an incorrect configuration?

Reply:
Sorry for my late reply! From what I know, the user can set options to configure IMAP. It is not required though.

But if he/she starts configuring it but does not complete all the fields, it is not incorrect but incomplete (= it won't work). And if he/she fills all the fields in but the configuration is wrong, it is incorrect (= it still does not work).

Reply:
Yes, that's it :)

Reply:
I'm sorry but I don't understand which warning notification will be displayed in case of an incorrect configuration. 😅 
Is it "IMAP is not installed on this server"?

Reply:
"IMAP is not installed", but how should we install it? Is "installing" and "configuring" the same? I'm lost sorry.🤣🤔

Reply:
@LouiseBonnard What do you think of: 'IMAP could not be installed on the server.'

Reply:
Very clear! I'd just add an element of solution to help users know what to do. For instance, _IMAP could not be installed on the server, check your configuration and try again._ or any other wording that could inform the merchants they need to check the information filled.

Reply:
Yes, sounds great!

Reply:
I updated the notifications here: https://github.com/PrestaShop/prestashop-specs/pull/187/commits/5ae453058a2427a7e25d8afdc239663edd275743
Is it ok for you @Julievrz ?

Reply:
Yes, perfect! thank you!